src/node_internals.h

```diff
@@ -271,9 +278,25 @@ class ThreadPoolWork {
 
   Environment* env() const { return env_; }
 
+  const char* type_name() {
+    switch (type_) {
+      case ThreadPoolWorkType::BLOB:
+        return "blob";
+      case ThreadPoolWorkType::CRYPTO:
+        return "crypto";
+      case ThreadPoolWorkType::ZLIB:
+        return "zlib";
+      case ThreadPoolWorkType::NODE_API:
+        return "node_api";
+      default:
+        return "unknown";
+    }
+  }
+
  private:
   Environment* env_;
   uv_work_t work_req_;
+  ThreadPoolWorkType type_;
```

Comment:
Why not just make this a `const char*` if that's the only way it's being accessed anyway?

Reply:
Done ! Thanks for the suggestion.